subsys/sd/sd_ops.h

```diff
@@ -10,7 +10,7 @@
 
 /*
  * Switches voltage of SD card to 1.8V, as described by
- * "Signal volatage switch procedure" in section 3.6.1 of SD specification.
+ * "Signal voltage switch procedure" in section 3.6.1 of SD specification.
```

Comment:
Thanks for catching this- while you are at it, it looks like the section is now named "Signal Voltage Switch Sequence". We should also drop the section number, that may change with revisions of the specification

Reply:
I was indeed looking at the wrong spec...apologies, shouldn't be commenting on GitHub so early in the morning 😆. Could you change "SD specification" to "SD host controller specification" to protect me from myself? I do want to refer the section 3.6.1 in the SD host controller simplified specification

Reply:
Already push another commit